Lesson4/Task4.py

```diff
@@ -0,0 +1,9 @@
+name = 'dmytro'
+inp_name = input('Enter your name: ')
+
+if inp_name.lower == name \
+        or inp_name.upper() \
+        or inp_name.capitalize():
+    print(True)
```

Comment:
This code is not working properly. Please check and fix it. Ping me if you have questions
```suggestion
if inp_name.lower == name \
        or inp_name.upper() \
        or inp_name.capitalize():
    print(True)
```

Reply:
Fix provided, @Masergio, review, please.